ecommerce-ui/src/App.js

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react';
+import RentalSelector from './RentalSelector.js';
+import ShoppingCart from './ShoppingCart.js';
+import './App.css';
+import './RentalSelector.css';
+import './ShoppingCart.css';
+import Airbnbs from './airbnbs.json';
+
+export default class App extends Component {
+  state = {
```

Comment:
Since you are not using an explicit constructor for anything, initializing state in this way is ok.